### DISCUSSION TITLE
Tabler UI Kit – We're Building an Open-Source Figma Design Kit!

### DISCUSSION THREAD

**Discussion by SilverFoxA** [tabler/tabler#1973] [General]
Hi Tabler team and community,

First of all, a huge thank you to the entire Tabler team for building and maintaining this fantastic open-source HTML Dashboard UI Kit! Your work has provided a valuable resource for developers and businesses, and it's truly appreciated.

As a designer and developer working on several UI-heavy projects, I noticed that one of the challenges for designers and businesses is the lack of a Figma file for Tabler, which can make the design prototyping process a bit difficult. To help solve this, we’re excited to announce that my team and I are working on creating a comprehensive Figma design board for Tabler. **The best part is that it will be open-source, just like Tabler!**

Our [Figma UI Kit](https://github.com/1Madgeek/tabler-ui-kit) will aim to mirror all the Tabler components, helping designers create high-fidelity prototypes and collaborate more seamlessly with developers. We believe this will benefit both the design and developer communities who use Tabler for their projects.

I will be dedicating a full-time designer from my company to this effort, and we aim to release the kit soon. In the spirit of open source, we'd love to invite any contributions or feedback from the community. Whether you’re a designer, developer, or just a fan of Tabler, we’d appreciate your insights or any support you might want to offer.

If you’re interested in contributing or collaborating, feel free to comment here, and we can coordinate!

Looking forward to giving back to this awesome community and continuing the open-source journey.

**Comment by dylan-tao** [comment]
> Hi Tabler team and community,
> 
> First of all, a huge thank you to the entire Tabler team for building and maintaining this fantastic open-source HTML Dashboard UI Kit! Your work has provided a valuable resource for developers and businesses, and it's truly appreciated.
> 
> As a designer and developer working on several UI-heavy projects, I noticed that one of the challenges for designers and businesses is the lack of a Figma file for Tabler, which can make the design prototyping process a bit difficult. To help solve this, we’re excited to announce that my team and I are working on creating a comprehensive Figma design board for Tabler. **The best part is that it will be open-source, just like Tabler!**
> 
> Our [Figma UI Kit](https://github.com/1Madgeek/tabler-ui-kit) will aim to mirror all the Tabler components, helping designers create high-fidelity prototypes and collaborate more seamlessly with developers. We believe this will benefit both the design and developer communities who use Tabler for their projects.
> 
> I will be dedicating a full-time designer from my company to this effort, and we aim to release the kit soon. In the spirit of open source, we'd love to invite any contributions or feedback from the community. Whether you’re a designer, developer, or just a fan of Tabler, we’d appreciate your insights or any support you might want to offer.
> 
> If you’re interested in contributing or collaborating, feel free to comment here, and we can coordinate!
> 
> Looking forward to giving back to this awesome community and continuing the open-source journey.

The official website has already released relevant illustrations, but it adopts the model of partial fee and small part free. Please mark my answer, thank you!
60+ sleek illustrations： https://tabler.io/illustrations

**Comment by janstadt** [comment]
Any update on this? All i see is a style guide and single structure figma design, no component breakout, no layout breakout.

**Reply by SilverFoxA** [reply]
@janstadt We will update the design board this month. Thank you.

**Comment by marietalia-ux** [comment]
Also very interested in this! Any updates?

**Reply by SilverFoxA** [reply]
@marietalia-ux We will update the design board this month. Thank you.

**Comment by HarryDrew** [comment]
Looks interesting

**Comment by janstadt** [comment]
@SilverFoxA has this initiative stalled out? I dont see any meaningful updates in the past 7 months.

**Reply by SilverFoxA** [reply]
@janstadt I'm sorry, we had some other commitments, which is why we couldn't work on it. I have assigned the team to update the board.

**Comment by SilverFoxA** [comment]
I'm excited to updated that we have released the initial version. It includes the following components:

1. Accordions
2. Alerts
3. Buttons
4. Cards
5. Badges
6. Modal 
7. Navigation
8. Pagination
9. Authentication Screens
10. Dropdowns
11. Avatars
12. Carousel
13. Data grid

<img width="1497" height="829" alt="Screenshot 2025-09-19 at 6 10 13 PM" src="https://github.com/user-attachments/assets/3a59be75-6e56-4b9c-9cf3-88941b717f0e" />

**Comment by SilverFoxA** [comment]
Update: We have added the form element components.

<img width="691" height="651" alt="Screenshot 2025-10-06 at 12 23 22 PM" src="https://github.com/user-attachments/assets/57091701-e8cb-4b3b-88de-794698fe154d" />
